tests/lmeval/test_lmeval.py

```
@@ -155,78 +159,68 @@ def test_lm_eval(self, test_data_file: str):
-            fewshot_random_seed=self.config.seed,
+        return self._eval_model_with_vllm(self.config.save_dir)
+
+    def _eval_model_with_vllm(self, model: str) -> dict:
+        run_file_path = os.path.join(test_file_dir, "run_lmeval.py")
+        logger.info("Run lm-eval in subprocess.Popen using python env:")
+        logger.info(VLLM_PYTHON_ENV)
+        # Ensure the venv's bin dir is on PATH so tools like ninja can be found
+        env = os.environ.copy()
+        venv_bin = os.path.dirname(VLLM_PYTHON_ENV)
+        env["PATH"] = venv_bin + os.pathsep + env.get("PATH", "")
+        result = subprocess.Popen(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)